requirements.txt

```diff
@@ -5,10 +5,14 @@ djangorestframework-simplejwt==5.3.1
 djoser==2.3.1
 django-filter==24.3
 django-debug-toolbar==4.4.6
+factory_boy==3.3.1
 gunicorn==23.0.0
 psycopg2-binary==2.9.10
 python-dateutil
 python-decouple==3.8
+pytest==8.3.4
+pytest-django==4.9.0
+pytest-factoryboy==2.7.0
```

Comment:
Why we need this package? `pytest-factoryboy==2.7.0`

Reply:
There wasn't a need for this package, I have uninstalled and removed it.